format-specs/geoparquet.md

```diff
@@ -76,21 +77,27 @@ GEOGCRS["WGS 84",
         MEMBER["World Geodetic System 1984 (G1674)"],
         MEMBER["World Geodetic System 1984 (G1762)"],
         MEMBER["World Geodetic System 1984 (G2139)"],
-        ELLIPSOID["WGS 84",6378137,298.257223563],
+        ELLIPSOID["WGS 84",6378137,298.257223563,
+            LENGTHUNIT["metre",1]],
         ENSEMBLEACCURACY[2.0]],
+    PRIMEM["Greenwich",0,
+        ANGLEUNIT["degree",0.0174532925199433]],
     CS[ellipsoidal,2],
-        AXIS["geodetic latitude (Lat)",north],
-        AXIS["geodetic longitude (Lon)",east],
-        UNIT["degree",0.0174532925199433],
+        AXIS["geodetic longitude (Lon)",east,
+            ORDER[1],
+            ANGLEUNIT["degree",0.0174532925199433]],
+        AXIS["geodetic latitude (Lat)",north,
+            ORDER[2],
+            ANGLEUNIT["degree",0.0174532925199433]],
     USAGE[
-        SCOPE["Horizontal component of 3D system."],
+        SCOPE["Not known."],
         AREA["World."],
         BBOX[-90,-180,90,180]],
-    ID["EPSG",4326]]
+    ID["OGC","CRS84"]]
 ```
 
-Due to the large number of CRSes available and the difficulty of implementing all of them, we expect that a number of implementations will at least start with only supporting a single CRS. To maximize interoperability we strongly recommend GeoParquet tool providers to always implement support for [EPSG:4326](https://epsg.org/crs_4326/WGS-84.html). 
-Users are recommended to store their data in EPSG:4326 for it to work with the widest number of tools. But data that is better served in particular projections can choose to use an alternate coordinate reference system. We expect many tools will support alternate CRSes, but encourage users to check.
+Due to the large number of CRSes available and the difficulty of implementing all of them, we expect that a number of implementations will at least start with only supporting a single CRS. To maximize interoperability we strongly recommend GeoParquet tool providers to always implement support for [OGC:CRS84](http://www.opengis.net/def/crs/OGC/1.3/CRS84). 
```

Comment:
There is a working `https` link. Let's use that instead.

https://www.opengis.net/def/crs/OGC/1.3/CRS84

Offtopic: I noticed that the XML definition files point to the `http` links (for example https://www.opengis.net/def/crs/OGC). Does anyone know why and if it's possible to update those XML files?

```xml
<identifiers xmlns="http://www.opengis.net/crs-nts/1.0" xmlns:gco="http://www.isotc211.org/2005/gco" xmlns:gmd="http://www.isotc211.org/2005/gmd" at="http://www.opengis.net/def/crs/OGC">
<identifier>http://www.opengis.net/def/crs/OGC/0</identifier>
<identifier>http://www.opengis.net/def/crs/OGC/1.3</identifier>
</identifiers>
```

Reply:
You mean that they are using http:// instead of https:// ? 

@ogcscotts should be able to help, or at least find the person to help.

Reply:
I am checking with the latest discussions and conversations in the OGC Naming Authority... more very soon.

Reply:
update: for OGC Registries, we have revised policy for https (https://github.com/opengeospatial/NamingAuthority/issues/108#issuecomment-1047735391)

for schemas.opengis.net, the Architecture Domain Working Group will be meeting in the next weeks and into June to work out a policy and implementation for http and https.

Reply:
Sounds good. Thanks!